data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/sink/SinkTopicConfig.java

```diff
@@ -31,6 +35,9 @@ public class SinkTopicConfig extends CommonTopicConfig implements TopicProducerC
     @JsonProperty("create_topic")
     private boolean isCreateTopic = false;
 
+    @JsonProperty("max_message_bytes")
```

Comment:
I have a similar question as above. Why can't we use the producer properties?

Reply:
@dlvenable please see above